luigi/rpc.py

```diff
@@ -144,35 +146,36 @@ def __init__(self, url='http://localhost:8082/', connect_timeout=None):
         else:
             self._fetcher = URLLibFetcher()
 
-    def _wait(self):
+    def _get_retry_decorator(self):
+        def retry_logging(retry_state):
+            logger.warning("Failed connecting to remote scheduler %r", self._url, exc_info=True)
+            logger.info("Retrying attempt %r of %r (max)" % (retry_state.attempt_number, self._rpc_retry_attempts))
+
         if self._rpc_log_retries:
-            logger.info("Wait for %d seconds" % self._rpc_retry_wait)
```

Comment:
I think it is better to keep this log. Never know what users would do with it.

Reply:
added wait log and fixed test in 8f36797